### ISSUE TITLE
Exception in production environment when trying to upload images after clicking the 'browse server' button

### ISSUE THREAD

**Issue by illegalnumbers** [jipiboily/monologue#298]
I get the following problem when I click 'browse server' on the image upload button in a production environment. I thought it was weird that the button to upload images even showed up since I didn't install the extension that allowed it.

> 
No webpage was found for the web address: http://www.illegalnumbers.com/ckeditor/pictures?CKEditor=post_content&CKEditorFuncNum=1&langCode=en

**Comment by jipiboily** [comment]
@illegalnumbers are you doing that with @msevestre's [file upload extension](https://github.com/msevestre/monologue_image_upload)?

**Comment by illegalnumbers** [comment]
Nope, I didn't have that installed. Guess I should've been more clear. What I meant by 'I thought it was weird that the button to upload images even showed up since I didn't install the extension that allowed it.' is that I had that button on the UI but I thought something like that shouldn't show up if it's not enabled or if that plugin isn't installed.

**Comment by jipiboily** [comment]
Ah, gotcha. Yeah, seems like a better UX to now have it and have a plugin adding it.

**Comment by illegalnumbers** [comment]
There might be some other extension specific items in the UI right now that could be removed too. If I get some time I'll look around for some examples.
